### PR TITLE
Addon-controls/a11y: Fix PARAM_KEY export for consistency

### PR DESCRIPTION
**Pull request by matheo**
@shilman I found some little details with room for improvement, so I just:

* Fixed the link of the valid globs
* Improved the debug output when the merge types mismatch
* Exported PARAM_KEY from a11y and controls as the others

### CODE REVIEW

File: lib/core/src/server/preview/to-require-context.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ const fixBadGlob = deprecate(
     return match.input.replace(match[1], `@${match[1]}`);
   },
   dedent`
-    You're specified an invalid glob, we've attempted to fix it, please ensure the glob you specify is a valid glob. See: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#correct-globs-in-main-js
+    You're specified an invalid glob, we've attempted to fix it, please ensure the glob you specify is a valid glob. See: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#correct-globs-in-mainjs
```
Comment (shilman) [inline lib/core/src/server/preview/to-require-context.ts:12]: I think this link is used twice in this file? Commenting from my phone so hard to check

**Review by shilman — APPROVED** [review]
LGTM with one caveat

File: lib/core/src/server/preview/to-require-context.ts (hunk 1 above)
Comment (matheo) [reply lib/core/src/server/preview/to-require-context.ts:12]: I guess the phone missed the colors of the diff hehe

Comment (shilman) [reply lib/core/src/server/preview/to-require-context.ts:12]: aha the other bad link is in `entries.js` a different file. can you fix that too?

Comment (matheo) [reply lib/core/src/server/preview/to-require-context.ts:12]: done!

**Pull request merged**